### ISSUE TITLE
[a11y] - missing region landmark for example widgets

### ISSUE THREAD

**Issue by transkamp** [open-pioneer/trails-openlayers-base-packages#428]
### Issue description

Open https://www.w3.org/WAI/ARIA/apg/patterns/landmarks/examples/region.html

widget or panel template on the map is missing the assignment of an html “region” landmark with an aria label. The content of the aria label can be identical to the heading.

DOM-Tree
![Image](https://github.com/user-attachments/assets/f017a9dc-82ea-4edc-9dcf-8d58379ca807)

A11Y-Tree
![Image](https://github.com/user-attachments/assets/52462156-8deb-4beb-8253-b6ad3cd0ddb9)

Example:
https://www.w3.org/WAI/ARIA/apg/patterns/landmarks/examples/region.html

### Validations

- [x] Read and follow our [code of conduct](https://github.com/open-pioneer/.github/blob/main/CODE_OF_CONDUCT.md)
- [x] Read the [contributing guidelines](https://github.com/open-pioneer/trails-starter/blob/main/docs/Contributing.md)
- [x] Check that there isn't already a similar issue

**Comment by mbeckem** [comment]
Added `role="region"` and `"aria-labelledby=..."` (points to heading) attributes.

See https://github.com/open-pioneer/trails-openlayers-base-packages/commit/7f146a12e978becf0deb8d0292ff80ca4b5e32bd
